src/job-exporter/src/nvidia.py

```diff
@@ -131,8 +131,12 @@ def parse_smi_xml_result(smi):
             volatile = ecc_errors[0].getElementsByTagName("volatile")
             if len(volatile) > 0:
                 volatile = volatile[0]
-                single = volatile.getElementsByTagName("single_bit")[0].getElementsByTagName("total")[0]
-                double = volatile.getElementsByTagName("double_bit")[0].getElementsByTagName("total")[0]
+                try:
+                    single = volatile.getElementsByTagName("single_bit")[0].getElementsByTagName("total")[0]
+                    double = volatile.getElementsByTagName("double_bit")[0].getElementsByTagName("total")[0]
+                except:
+                    single = volatile.getElementsByTagName("sram_uncorrectable")[0]
```

Comment:
Could you please add some comments here so reader will know what's going on here.
I also suggest you add some unit test like [here](https://github.com/Microsoft/pai/blob/master/src/job-exporter/test/test_nvidia.py#L47). You can add sample nvidia-smi output in data directory.